docs/api-reference/io-columns-ranking.md

```diff
@@ -1,5 +1,5 @@
 ### Input and Output Columns
-The input label column data must be <xref:System.Single> and input group column
+The input label column data must be <xref:System.Single> or [key-typed](xref:Microsoft.ML.Data.KeyDataViewType). and input group column
```

Comment:
>input group column [](start = 117, length = 18)

rowGroupColumnName  is name of parameter in fasttree in lightgbm. Maybe Row group input column?

Reply:
@natke thoughts?

Reply:
Actually what does that last part of the sentence mean?

Reply:
There are two inputs ranker needs, one is features column and second is group id.